curriculum/challenges/english/25-front-end-development/lecture-importance-of-semantic-html/672995ac85fd943657c2ede5.md

```diff
@@ -10,6 +10,39 @@ dashedName: when-should-you-use-the-strong-element-over-the-bring-attention-to-e
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+When should you use the strong element over the bring attention to element?
+
+The "bring attention to" element, `b`, is commonly used to highlight keywords in summaries, or product names in reviews. Usually, browsers display this text in boldface. Here's an example using the `b` element to highlight product names in this review:
+
+```html
+<p>
+  We tested several products, including the <b>SuperSound 3000</b> for audio
+  quality, the <b>QuickCharge Pro</b> for fast charging, and the
+  <b>EcoClean Vacuum</b> for cleaning. The first two performed well, but the
+  <b>EcoClean Vacuum</b> did not meet expectations.
+</p>
+```
+
+The browser renders these parts of the text as bold text. This visual emphasis will draw readers attention to the product names.
+
+If you need to emphasize the importance of the text, you should use the `strong` element instead of the `b` element.
+
+`strong` is a semantic HTML element that emphasizes text that is crucial, or urgent. This is an example where the `strong` element is used to label a very important warning about the potential allergic reactions that customers may have to a product:
+
+```html
+<p>
+  <strong>Warning:</strong> This product may cause allergic reactions.
+</p>
+```
+
+The `strong` element communicates that sense of urgency.
+
+Visually both are very similar, because they are both rendered as bold by default. But their meanings are quite different. While the bring attention to element only draws attention to the text, without indicating the higher level of importance, the `strong` element does more than that. It conveys a sense of importance, or urgency. This is their main difference. 
```

Comment:
```suggestion
Visually both are very similar, because they are both rendered as bold by default. But their meanings are quite different. While the "bring attention to" element only draws attention to the text, without indicating the higher level of importance, the `strong` element does more than that. It conveys a sense of importance, or urgency. This is their main difference. 
```

Might as well do them all if we do any?